### PR TITLE
Fixes #5 Use port 10025 and postfix image for tests

### LINKED ISSUE #5: better testing workflow
* [x] do not use host's port 25 so the tests won't conflict with possible host's sendmail/postfix
* [x] use locally built image for testing, not the one in Docker Hub

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

Pull Request fixes #5 .
Use port 10025 and image name postfix for testing proposes.

### CODE REVIEW

File: tests/config.yaml [hunk 1]
```diff
@@ -12,9 +12,9 @@ testdependecies:
         - nc
 module:
     docker:
-        start: "docker run -e MYHOSTNAME=localhost -p 25:10025"
+        start: "docker run -e MYHOSTNAME=localhost -p 10025:10025"
         source: https://github.com/container-images/postfix.git
-        container: docker.io/modularitycontainers/postfix
+        container: postfix
```
Comment (TomasTomecek) [inline tests/config.yaml]: will mtf try to pull the image by default?

Comment (phracek) [inline tests/config.yaml]: MTF pulls the image specified in config.yaml. In this case postfix which should be mentioned in `docker images` no from docker hub. https://github.com/fedora-modularity/meta-test-family/blob/devel/moduleframework/helpers/container_helper.py#L146 If I understand right.

**Comment by LorbusChris** [comment]
I am wondering whether you should be able to run the process in a containerized and non-containerized fashion simultaneously, as you're not supposed to be running the same containerized process twice (per Modularity Docs / Buildung Modules / Standalone Container).

If so, changing the port is probably right. 

However, if in Modularity this were not officially supported/encouraged, I'd vote against it, as everyone would postfix expect to run on Port 25 (and you woulnt have the native process running there either in that case).

Also what about the Submission port 587? If we we're to use TLS it needs exposuret, too.
Maybe use standard ports inside the container and only expose them on alternate ports?

**Comment by TomasTomecek** [comment]
This should be solely about testing: you are always free to run it on whatever ports you wish.

**Comment by TomasTomecek** [comment]
@phracek thanks! tested locally, works fine; `URL="docker=postfix"` seems to do the trick

@LorbusChris Chris, if you have any further questions or concerns, feel free to open a new issue, I am going to merge this one.

**Pull request merged**